R/po_scan.R

```diff
@@ -0,0 +1,32 @@
+po_scan <- function(dir = ".", custom_translation_functions = list(), verbose = TRUE, style = c("base", "explicit")) {
+  style <- match.arg(style)
+
+  message_data <- get_message_data(dir,
+    custom_translation_functions = custom_translation_functions,
+    verbose = verbose,
+    include_conditions = style == "base",
+    include_conditions_f = style == "base",
```

Comment:
This adds automatic support for `mesasgef()` and friends to the "base" style. I think this is low-risk because it will only affect packages that actually define `messagef()` and friends and it seems unlikely that these would be used for non-potools purposes. But if you think that's too risky, we could add another style, maybe "base-strict"?

Reply:
I've been thinking we should nudge R-devel to include these wrappers actually, given that using `...` in translated messages actually encourages poor practice / difficult-to-translate strings. So including them is a good choice I think.